internal/gateway/getbgpneighbours.go

```diff
@@ -0,0 +1,49 @@
+package gateway
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+
+	"github.com/spf13/cobra"
+)
+
+func (c *Client) GetBgpNeighbours() *cobra.Command {
+	var bgpNeighbourID string
+
+	// getGwayBgpDynamicNeighbourCmd represents command to get Metal Gateway Dynamic Neighbour by ID.
+	getGwayBgpDynamicNeighbourCmd := &cobra.Command{
+		Use:   `get-bgp-dynamic-neighbours`,
+		Short: "Gets a BGP Dynamic Neighbour",
+		Long:  "Gets the BGP Dynamic Neighbour for the metal gateway with the specified ID",
```

Comment:
I think `metal gateway` shouldn't be mentioned here?

Reply:
Removed